src/components/App.js

```diff
@@ -1,8 +1,15 @@
 import React from "react";
+import { Route } from "react-router-dom";
+import { SkillsContainer } from "./SkillsContainer";
 
 export const App = () => {
   return (
-    <h1>Star Wars RPG Companion</h1>
+    <>
+      <h1>Star Wars RPG Companion</h1>
+      <Route path="/skills">
+        <SkillsContainer />
```

Comment:
Nitpick: I think SkillsContainer could be a bit more semantic. Could we do SkillsPage since were using skills-container for the class name?

Reply:
Agreed, I think SkillsPage makes more sense here. 👍